resolwe_bio/tests/processes/test_variant_calling.py

```diff
@@ -1,5 +1,6 @@
 # pylint: disable=missing-docstring
```

Comment:
Maybe we should move this from `.../tests/processes/...` to `.../tests/workflows/...`.